auth.proto

```diff
@@ -223,3 +224,19 @@ message DeleteEntityPoliciesReq{
   string entity_type = 1;
   string id          = 2;
 }
+
+message VerifyConnectionsReq {
+  repeated string thing_ids = 1;
+  repeated string group_ids = 2;
```

Comment:
For Verify-Connection HTTP request , it request requires `channel_ids` field in request body 

```bash
curl --location 'http://localhost/things/verify-connections' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer <TOKEN>' \
--data '{
    "thing_ids": ["92a70298-8279-41f1-ae7e-0ecebe6b6f4a"],
    "channel_ids": ["cd866cf4-e671-4db9-b181-1e803ea756ba"]
}'
```

And HTTP response also have like channel_id field 
```json
{
    "status": "all_connected",
    "connections_status": [
        {
            "channel_id": "cd866cf4-e671-4db9-b181-1e803ea756ba",
            "thing_id": "92a70298-8279-41f1-ae7e-0ecebe6b6f4a",
            "status": "connected"
        }
    ]
}
```

For gRPC request Body is required `group_ids` in request body 
```json
{
    "thing_ids": ["92a70298-8279-41f1-ae7e-0ecebe6b6f4a"],
    "group_ids": ["cd866cf4-e671-4db9-b181-1e803ea756ba"]
}
```
and gRPC response have `channel_id` in response body
```json
{
    "status": "all_connected",
    "connections_status": [
        {
            "channel_id": "cd866cf4-e671-4db9-b181-1e803ea756ba",
            "thing_id": "92a70298-8279-41f1-ae7e-0ecebe6b6f4a",
            "status": "connected"
        }
    ]
}
```

What do you think about making the gRPC request field `group_ids` to `channel_ids`
In order to align with HTTP API. and from user prespective channels and groups are different entity

Reply:
Let me change from `group_ids` to `channel_ids` to remove the confusion